### ISSUE TITLE
Render Bar Charts

### ISSUE THREAD

**Issue by PSeitz** [bheisler/criterion.rs#453]
By default the thoughpout generates line charts. 
In my use case bar charts would be more approriate, since there is no sequence of input size.

**Comment by bheisler** [comment]
Hey, thanks for the suggestion!

I think this would be a small improvement for a niche use case and so I don't think it makes sense to add more API surface to allow benchmarks to configure it. I'm going to close this for now, but I'd be happy to revisit in the future if it turns out to be a more common need than I think it is.

**Comment by PSeitz** [comment]
It was requested here: https://www.reddit.com/r/rust/comments/kaj50o/lz4_flex_041_matches_now_the_cpp_lz4_reference/gfbjac7/?context=3
Number of up-votes suggests there is a clear opinion about this.

In my use case this is the default, because I usually benchmark against different kinds of input.

**Comment by bheisler** [comment]
Yeah, okay. It might not be a priority but I'll reopen this and probably get around to it eventually.

**Comment by novacrazy** [comment]
Bar charts would definitely be of use for comparisons with different data sets not on a line.
